.github/workflows/bitcoin-tests.yml

```diff
@@ -82,12 +82,13 @@ jobs:
           - tests::nakamoto_integrations::vote_for_aggregate_key_burn_op
           - tests::nakamoto_integrations::follower_bootup
           - tests::signer::stackerdb_dkg
-          - tests::signer::stackerdb_sign
+          - tests::signer::stackerdb_sign_request_rejected
```

Comment:
Note: this is just a rename as the test does not actually perform sign anymore (changed in a different PR)